admin/js/enforms.js

```diff
@@ -1,5 +1,185 @@
 /* global ajaxurl, Backbone, _ */
 
+const enformTemplates = {
+  enSelectedField: data => {
+    const __ = wp.i18n.__;
```

Comment:
Can't we declare this only once at the top of the page?

Reply:
I'm a tiny bit afraid to declare things in global context :grimacing: